src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+---
+title: Interact with NFC devices on the web
+subhead: Reading and writing to NFC tags is now possible on the mobile web.
+authors:
+  - beaufortfrancois
+date: 2020-02-12
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A photo of NFC tags
+description: |
+  Reading and writing to NFC tags is now possible on the mobile web.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - progressive-web-apps
+---
+
+{% Aside %}
+Web apps should be able to do anything native apps can. The [Capabilities
+project](https://developers.google.com/web/updates/capabilities), of which Web
+NFC is only a part, aims to do just that. To learn about other capabilities and
+to keep up with their progress, follow [Unlocking new capabilities for the
+web](https://developers.google.com/web/updates/capabilities).
+{% endAside %}
+
+## What is Web NFC? {: #what }
+
+Web NFC provides sites the ability to read and write to NFC tags when they are
+brought in close proximity to the user's device (usually 5-10 cm, 2-4 inches).
+The current scope is limited to NDEF, a lightweight binary message format that
+works across different tag formats.
+
+{% Aside 'gotchas' %}
+Web NFC is limited to NDEF because the security properties of reading and
+writing NDEF data are more easily quantifiable. Low-level I/O operations (e.g.
+ISO-DEP, NFC-A/B, NFC-F) and Host-based Card Emulation (HCE) are not supported.
+{% endAside %}
+
+<figure class="w-figure">
+  <img src="./nfc-operation-diagram.png" alt="Phone powering up an NFC tag to exchange data">
+  <figcaption class="w-figcaption">Diagram of an NFC operation</figcaption>
+</figure>
+
+## Suggested use cases {: #use-cases }
+
+Examples of sites that may use Web NFC include:
+- Museums and art galleries can display more information about a display when
+  the user touches their device to a NFC card near the exhibit.
+- Inventory management sites can read or write data to the NFC tag on a
+  container to update information on its contents.
+- Conference sites can use it to scan NFC badges during the event.
+- Sites can use it for sharing initial secrets needed for device or service
```

Comment:
Something about reading preconfigured settings/modes, like an iot control site and the likes

Reply:
Please provide a full sentence to be included.